src/purchase_listinator/endpoints/http/purchase_list.clj

```diff
@@ -32,7 +34,8 @@
             (let [user-id (adapters.misc/string->uuid user-id)]
               (flows.purchase-list/create name user-id datomic)))
           misc.http/->Error
-          misc.http/->Success))
+          #(-> (adapters.out.purchase-list/internal->wire %)
```

Comment:
Applying the adapter so the function respects the expected schema output.

Reply:
Almost the same here :)